CardForm/src/test/java/com/braintreepayments/cardform/view/CardFormTest.java

```diff
@@ -1059,25 +1138,29 @@ public void handleCardIOResponse_doesNotSetExpirationDateIfExpirationDateNotRequ
         assertTrue(mCardForm.findViewById(R.id.bt_card_form_cvv).hasFocus());
     }
 
-    private void setRequiredFields(boolean cardNumberRequired, boolean expirationRequired, boolean cvvRequired,
+    private void setRequiredFields(boolean cardNumberRequired, boolean expirationRequired, boolean cvvRequired, boolean cardholderNameRequired,
```

Comment:
The params for these test methods should probably be reordered to have cardholder name first so it's clearly when reading and writing tests which field you mean to set.